### ISSUE TITLE
Problems arising from labeling samples with nan value in the processing of adata

### ISSUE THREAD

**Issue by steamed-stuffed-bun** [ChengmingZhang-CAS/CauFinder-main#1]
I'm executing the file LUAS_human_analysis.py and trying to retrain the model, and I'm finding that the calculated loss is the null value. The reason for this is that the calculated dpd_loss is null, whereas the source code for calculating the dpd_loss shows that:
`dpd_loss = F.binary_cross_entropy_with_logits(org_logit.squeeze(), y, pos_weight=pos_weight, reduction='none')`
the y refers to the labeling of the samples, and from the function load_luas_human_adata in the utils.py script that you have provided, the author labels the samples in the intermediate state with a value of nan, which results in an error when training the model. This results in an error when training the model:
![error](https://github.com/user-attachments/assets/0e3ecb5c-f576-41ef-9adb-7cff524df23d)
How can I fix this error when training the model?

**Comment by ChengmingZhang-CAS** [comment]
Thank you for pointing this out! You are right that the issue stems from labeling the samples in the intermediate state as `NaN`, which causes an error when calculating `dpd_loss` during model training. 

There are two possible solutions:  
1. You can use `adata_filter` for training to ensure that all `y` values are labeled, which will avoid the `NaN` issue.  
2. Alternatively, if you want to retain the samples in the intermediate state, you can manually set the intermediate state labels to 0.5, which will prevent errors during the loss calculation.

I've updated the code to fix the issue with `NaN` labels.
